operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -137,6 +137,27 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
+      $._config.forwarding_enabled,
+      $.row('Distributor Forwarding')
+      .addPanel(
+        $.panel('Requests / sec') +
+        $.successFailurePanel(
+          'sum(rate(cortex_distributor_forward_requests_total{%(distributorMatcher)s}[$__rate_interval]))' % $.jobMatcher($._config.job_names.distributor),
```

Comment:
Here we're expected to show the rate of successes, not total requests. Does `cortex_distributor_forward_requests_total` only track successes or all attempts? If all attempts, than you probably want to do `cortex_distributor_forward_requests_total - cortex_distributor_forward_errors_total`.

Reply:
Nice catch, thx. 
I'm not quite sure what's the best way to do that with two rates, how about this: 617be972da